apps/etterlatte-saksbehandling-ui/client/src/components/person/aktivitet/Aktivitet.tsx

```diff
@@ -0,0 +1,139 @@
+import { isSuccess, mapResult, Result } from '~shared/api/apiUtils'
+import { SakMedBehandlinger } from '~components/person/typer'
+import React, { ReactNode, useEffect, useState } from 'react'
+import { BodyShort, Box, Heading, Table, VStack } from '@navikt/ds-react'
+import {
+  IAktivitetspliktVurderingNy,
+  tekstAktivitetspliktUnntakType,
+  tekstAktivitetspliktVurderingType,
+} from '~shared/types/Aktivitetsplikt'
+import { useApiCall } from '~shared/hooks/useApiCall'
+import { hentAktivitspliktVurderingForSak } from '~shared/api/aktivitetsplikt'
+import { formaterDato, formaterDatoMedFallback } from '~utils/formatering/dato'
+import Spinner from '~shared/Spinner'
+
+export const Aktivitet = ({ sakResult }: { sakResult: Result<SakMedBehandlinger> }): ReactNode => {
+  const [hentet, hent] = useApiCall(hentAktivitspliktVurderingForSak)
+  const [vurdering, setVurdering] = useState<IAktivitetspliktVurderingNy>()
+
+  useEffect(() => {
+    if (isSuccess(sakResult)) {
+      hent({ sakId: sakResult.data.sak.id }, (aktivitetspliktVurdering) => {
+        setVurdering(aktivitetspliktVurdering)
+      })
+    }
+  }, [])
+
+  return (
+    <Box padding="8" maxWidth="70rem">
+      <Heading size="medium" spacing>
+        Aktivitet
+      </Heading>
+
+      {mapResult(hentet, {
+        pending: <Spinner label="Henter vurderinger..." />,
+        error: (error) => <BodyShort>{error.detail || 'Kunne ikke hente vurderinger'}</BodyShort>,
```

Comment:
BodyShort for error er kanskje ikke helt ideelt

Reply:
Det er sant!